src/components/login/LoginPage.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react'
+import Layout from 'components/layout/Layout'
+import { Typography, Container, Grid, TextField, Button } from '@material-ui/core'
+
+export default function LoginPage() {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+
+  function handleSubmit(event) {
```

Comment:
@ani-kalpachka You might wanna typehint the `event` with `event: FormEvent` after importing it from `react`